opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -423,6 +427,8 @@ def start_span(
             attributes: The span's attributes.
             links: Links span to other spans
             start_time: Sets the start time of a span
+            auto_update_status: Defines if the status should be updated
+                automatically when the span finishes
```

Comment:
This is too vague. How does it update?

I suggest: "Whether the span status should automatically be set to an error when an exception leaves the `with`-block where the span is used. A manually set status overrides this, even if set before the exception."

Reply:
Fixed too!